packages/admin-ui/src/pages/support/formaters/autoDiagnoseTexts.ts

```diff
@@ -87,15 +87,16 @@ export function openPorts({
 }): DiagnoseResultOrNull {
   if (isValidating) return { loading: true, msg: "Loading system info..." };
   if (!dappnodeParams) return null;
-  const { alertToOpenPorts } = dappnodeParams;
+  const { alertToOpenPorts, publicIp } = dappnodeParams;
   return {
     ok: !alertToOpenPorts,
     msg: alertToOpenPorts
       ? "Ports have to be opened and there is no UPnP device available"
       : "No ports have to be opened OR the router has UPnP enabled",
     solutions: [
       "If you are capable of opening ports manually, please ignore this error",
-      "Your router may have UPnP but it is not turned on yet. Please research if your specific router has UPnP and turn it on"
+      "Your router may have UPnP but it is not turned on yet. Please research if your specific router has UPnP and turn it on",
+      `Check your ports: https://check-host.net/check-udp?host=${publicIp}%3A1194`
```

Comment:
Be a bit more helpful in the message. Explain what this page does, why is it useful for this problem and even consider adding a second link with a forum guide explaining how to do it.